.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Summary**
+[A clear and concise description of what the bug is]
+
+**To Reproduce**
+1. [Steps to reproduce]
```

Comment:
```suggestion
[Steps to reproduce]
```

Reply:
I wanted to suggest providing steps as a numbered list, if it makes sense.